rules/joinbox-custom.js

```diff
@@ -3,14 +3,14 @@ module.exports = {
         // refs: https://github.com/joinbox/guidelines/blob/master/styleguide/javascript.md
         // 7.1 allow function decalrations
         'func-style': ['off', 'declaration'],
-        // 7.1 allow only anaonymous function declarations
+        // 7.1 allow only anonymous function declarations
         'func-names': ['error', 'never'],
-        // 17.1 doesnt need a custom config rule
-        // 18.4 alle extensive blank lines
+        // 17.1 doesn't need a custom config rule
+        // 18.4 allow extensive blank lines
         'padded-blocks': 'off',
         'no-multiple-empty-lines': 'off',
         // 23.4
-        'no-underscore-dangle': ['error', { allowAfterThis: true }],
+        'no-underscore-dangle': 'off',
```

Comment:
Hmm actually it would make sense to keep it as a warning! The `allowAfterThis` parameter makes absolutely sense to me: If you access a method/property with a leading underscore without being in the context of a class instance (indicated by this) you are accessing stuff you should not!

Reply:
Btw: if you work with Mongodb stuff you could disable the corresponding linter rule for the whole file...